### PR TITLE
RDoc for converters

### PR DESCRIPTION
**Pull request by BurdetteLamar**
This PR had to be this big because of cross-referencing (links) and to avoid redundancy.

### CODE REVIEW

File: lib/csv.rb [hunk 1]
```diff
@@ -440,54 +440,188 @@
...
+#   - A \Proc field converter.
+#   - An \Array of field converter names.
 #
-# Converter +:integer+ converts each field that +Integer()+ accepts:
+# Display:
+#   CSV::Converters.each_pair do |name, value|
+#     if value.kind_of?(Proc)
+#       p [name, value.class]
+#     else
+#      p [name, value]
```
Comment (kou) [inline lib/csv.rb:609]:
```suggestion
#       p [name, value]
```

**Comment by kou** [comment]
Thanks.

**Comment by BurdetteLamar** [comment]
@kou, does this eventually get merged into ruby/ruby?

**Comment by kou** [comment]
Yes.
But we need to consider where we should put separated documents in ruby/ruby before we put our changes to ruby/ruby.

**Pull request merged**